exonum/src/storage/sparse_list_index.rs

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! An implementation of array list of items with spaces.
+//! An implementation of an array list of items with spaces.
```

Comment:
See the comment for `list_index` module.

Reply:
I've left a short introductory sentence, as I've added a similar introduction to other modules of this section, so perhaps it would be good if all modules had this introduction.